### ISSUE TITLE
Nested numbered list indexes

### ISSUE THREAD

**Issue by gotschmarcel** [uranusjr/macdown#458]
It would be great if nested numbered lists would result in indexes like: 1.1, 1.1.1, ... as it is in other text applications like MSWord.

**Comment by Jmuccigr** [comment]
I'm betting you can do this with css and section numbering.

**Comment by uranusjr** [comment]
Yes, custom CSS would be needed since it is not natively supported by HTML, thus impossible in Markdown alone.

It is an interesting idea though. I wonder if it is possible to achieve this with some custom syntax…

**Comment by Jmuccigr** [comment]
Look at this: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Lists_and_Counters/Using_CSS_counters
